Tests/FunctionalTests/DependencyResolutionTests.swift

```diff
@@ -18,12 +18,14 @@ import _InternalTestSupport
 import Workspace
 import XCTest
 
+import enum TSCUtility.Git
```

Comment:
thought: I wonder if we should vender the `Git` symbol in SwiftPM so we remove a dependency on TSC.  I guess it's a discussion for another time.